week5/joshcho_react/src/ProductTable.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import ProductCategoryRow from "./ProductCategoryRow";
+import ProductRow from "./ProductRow";
+
+const ProductTable = ({
+  products,
+  filterText,
+  inStockOnly,
+  deleteProduct,
+  updateProduct,
+}) => {
+  const rows = [];
+  let lastCategory = null;
+
+  products
+    .sort((a, b) => a.category > b.category)
```

Comment:
.sort((a, b) => a.category.localeCompare(b.category))

여기 정렬 부분이 잘못되어서 말씀하신 오류가 발생합니다.

숫자를 sort 하는거라면 위와 같은 방법도 좋지만 문자열을 sort 해야하기 때문에 

```suggestion
    .sort((a, b) => a.category.localeCompare(b.category))
```
문자열을 sort 할 수 있는 localeCompare와 같은 메서드를 사용하시면 해결됩니다!